src/index.ts

```diff
@@ -27,15 +27,29 @@ export const beautifier: Beautifier = {
       parseVersion: [/version (.*) by/, /PHP CS Fixer (\d+\.\d+\.\d+)/],
     },
   ],
-  beautify({ text, dependencies, filePath }: BeautifierBeautifyData) {
+  resolveConfig: ({ filePath, projectPath }) => {
+    const configFiles: string[] = [".php_cs", ".php_cs.dist"];
+    return findFile({ finishPath: projectPath, startPath: filePath, fileNames: configFiles }).then(configFile => {
+      return Promise.resolve({ filePath: configFile });
+    })
+    .catch((err) => {
+      // tslint:disable-next-line
+      console.log(err);
+      return Promise.resolve({});
+    });
+  },
+  beautify({ text, dependencies, filePath, beautifierConfig }: BeautifierBeautifyData) {
     const phpCsFixer = dependencies.get<ExecutableDependency>("PHP-CS-Fixer");
     const basePath: string = os.tmpdir();
+    const config = beautifierConfig && beautifierConfig.filePath ? `--config=${beautifierConfig.filePath}` : "--rules=@PSR2";
```

Comment:
I wonder if a better default than `@PSR2` is no configuration?

https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/php-cs-fixer.coffee#L69-L71

Reply:
If no rules or config is passed, then by default the PSR1 and PSR2 rules are used.